app/Models/Event.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Support\Collection;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+
```

Comment:
```suggestion

/*
 * Class Event.
 *
 * @property string $title
 * @property Carbon $start_datetime
 * @property Carbon $end_datetime
 * @property Carbon|null $created_at
 * @property Carbon|null $updated_at
 */
```

Це називається PHPDoc блок, можеш за це почитати: https://docs.phpdoc.org/guide/guides/docblocks.html